refarch-apigateway/frontend/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "refarch-apigateway-frontend",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview",
+    "test:unit": "vitest run",
+    "lint": "prettier . --check && eslint . && vue-tsc --noEmit --skipLibCheck",
+    "fix": "prettier . --write && eslint . --fix"
+  },
+  "dependencies": {
+    "@mdi/font": "7.3.67",
+    "@muenchen/appswitcher-vue": "^2.0.0",
+    "moment": "^2.29.4",
+    "pinia": "^2.1.7",
+    "roboto-fontface": "^0.10.0",
+    "vue": "^3.3.7",
+    "vue-router": "^4.2.5",
+    "vuetify": "^3.3.23"
+  },
+  "devDependencies": {
+    "@babel/types": "^7.23.0",
+    "@ianvs/prettier-plugin-sort-imports": "^4.1.1",
+    "@muenchen/prettier-codeformat": "^1.0.1",
+    "@types/jsdom": "^21.1.4",
+    "@types/node": "^18.18.8",
+    "@vitejs/plugin-vue": "^4.4.0",
+    "@vue/eslint-config-prettier": "^8.0.0",
+    "@vue/eslint-config-typescript": "^12.0.0",
+    "@vue/test-utils": "^2.4.1",
+    "eslint": "^8.52.0",
+    "eslint-plugin-vue": "^9.18.1",
+    "eslint-plugin-vuetify": "^2.0.5",
+    "jsdom": "^22.1.0",
+    "prettier": "^3.0.3",
+    "typescript": "^5.2.2",
+    "unplugin-fonts": "^1.0.3",
+    "vite": "^4.5.0",
+    "vite-plugin-vuetify": "^1.0.2",
+    "vitest": "^0.34.6",
+    "vue-tsc": "^1.8.22"
+  }
```

Comment:
In my opinion we should start pinning the dependency versions. This is especially useful if we start versioning our templates later on. Renovatebot can help managing those dependencies on patch level as well.
Pinning deps will sync it with the behaviour in Java based code. Here dependencies are also pinned.

Reply:
By 'pinning' you mean we should always use the exact version without `^` or `~`? That would mean more LCM-effort for everyone

Reply:
That's true. But not pinning it leads to inconsistent behaviour and might even introduce side effects or problems when a newer version is used without explicitly defining it. We can discuss this at a later point as well. I will create a issue for it.